subgraphs/_eslint-rules/no-non-standard-filenames.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  // see https://github.com/messari/subgraphs/issues/1484
+  // folder and file names must be in snake_case or kebab-case; leading "_" allowed.
+  meta: {
+    docs: {
+      description:
+        "filename and folder name must be in snake_case or kebab-case",
```

Comment:
If we stick to this guide: https://google.github.io/styleguide/tsguide.html#identifiers-imports, we should use snake_case for file names too?

Currently the file names conventions used include camelCase, PascalCase (abis) and single word filenames (ts scripts).

removed leading "_"

Reply:
open to discussion on this, but since we already have an established pattern i would think we should stick with that and not introduce a new filename kind.

Reply:
Also, we could only check `.ts` files for camelCase. And `.json` files for PascalCase. That way we don't accidentally throw errors for files we don't care about their naming

Reply:
ok, added.